mne/io/eeglab/eeglab.py

```diff
@@ -445,3 +445,46 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _create_events_from_eeglab_raw(fname, event_id=dict()):
```

Comment:
I was thinking having it work like e.g. the Brainvision reader, which just returns a `raw` with a stim channel. And this would remain a private method that is called internally by read_raw_eeglab to fill the stim channel with values.